cmd/abci-cli/abci-cli.go

```diff
@@ -143,14 +143,20 @@ func main() {
 				},
 			},
 		},
+		{
+			Name:  "test",
+			Usage: "Runs the integration tests",
+			Action: func(c *cli.Context) error {
+				return cmdTest(app, c)
+			},
+		},
```

Comment:
we're switching to cobra in #117, so this'll need to be re-worked. Maybe base this PR on that one?

Reply:
I’ll rebase if #117 is merged earlier.